src/client/app/types/redux/state.ts

```diff
@@ -16,15 +15,15 @@ import { CurrentUserState } from './currentUser';
 import { UnsavedWarningState } from './unsavedWarning';
 import { UnitsState } from './units';
 import { ConversionsState } from './conversions';
-import { OptionsState } from './options';
+import { AppState } from 'redux/slices/appStateSlice';
 
 export interface State {
+	appState: AppState;
 	meters: MetersState;
 	readings: {
 		line: LineReadingsState;
 		bar: BarReadingsState;
 		compare: CompareReadingsState;
```

Comment:
Just curious why 3D was removed from here.

Reply:
This is old state, which is only used in maps. 3d was the first ti implement RTKQ, so was remove here. But this all should go soon, so not of much importance. Resolving convo.